### PR TITLE
C1s14 know how trophies

### PR DESCRIPTION
**Pull request by adamggg**
Scenario Reviewer: @KareemAliRahman
Implementation Reviewer: @NourhanMohamed
UML Reviewer: @Khloud-Khalid
Documentation Reviewer: @zoheiry

### CODE REVIEW

**Comment by adamggg** [comment]
#171

File: arability/app/views/games/_show_trophies.html.erb [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<div class="award-zone" id="award-list">
+  <div class="won">
+    <% if I18n.locale == :en %>
+      <a class="pull-right btn btn-success" style="padding: 5px; margin-right: 12px; margin-top: 28px;" onclick="backToGame()">Back</a>
+    <% else %>
+      <a class="pull-left btn btn-success" style="padding: 5px; margin-left: 28px; margin-top: 28px;" onclick="backToGame()">رجوع </a>
+    <% end %>
+    <h3 class="title-head"><%= t(:won_trophies_title) %></h3>
+    <% if @won_trophies.size == 0 %>
```
Comment (NourhanMohamed) [inline arability/app/views/games/_show_trophies.html.erb]: use .blank?

File: arability/app/views/games/_show_trophies.html.erb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            <%= t(:score) %> <%= trophy.score %>
+          </div>
+        </div>
+      <% end %>
+    <% end %>
+  </div>
+
+  <div class="not-won">
+    <h3 class="title-head"><%= t(:not_won_trophies_title) %></h3>
+    <% if @not_won_trophies.size == 0 %>
```
Comment (NourhanMohamed) [inline arability/app/views/games/_show_trophies.html.erb]: use .blank?

**Comment by krmali** [comment]
@adamggg , solve conflicts

**Comment by adamggg** [comment]
@KareemAliRahman Please merge =)

**Pull request merged**